test/test_utils/sagemaker.py

```diff
@@ -88,7 +88,8 @@ def generate_sagemaker_pytest_cmd(image, sagemaker_test_type):
     :param sagemaker_test_type: local or remote test type
     :return: <tuple> pytest command to be run, path where it should be executed, image tag
     """
-    reruns = 4
+    reruns = 1
```

Comment:
Does this override the number of reruns we set for individual SM tests? If so, is `reruns = 1` sufficient?

Reply:
updated the description on this.